src/context.cpp

```diff
@@ -34,11 +34,10 @@
 #include "emitter.hpp"
 
 namespace Sass {
-  using namespace Constants;
-  using namespace File;
-  using namespace Sass;
-  using std::cerr;
-  using std::endl;
+
+  using std::unordered_map;
+  using File::make_canonical_path;
```

Comment:
Maybe we can get rid of the `File` namespace at all (if it makes making bindings more difficult...)?